src/main/scala/com/microsoft/partnercatalyst/fortis/spark/analyzer/TwitterAnalyzer.scala

```diff
@@ -13,7 +13,7 @@ class TwitterAnalyzer extends Analyzer[TwitterStatus] with Serializable
       eventtime = item.getCreatedAt.getTime,
       body = item.getText,
       title = "",
-      externalsourceid = item.getSource,
+      externalsourceid = item.getUser.getScreenName,
```

Comment:
`item.getUser.getScreenName` returns the twitter user's handle, e.g. `@HamillHimself`